README.md

```diff
@@ -31,3 +30,102 @@ After that create a `.eslintrc.json` file and add the following:
+
+### The changes are smaller than they appear:
+
+#### One environment was removed:
+
+- Remove `mocha` from `env` because this belongs into [eslint-config-signavio-test](https://github.com/signavio/eslint-config-signavio-test)
+
+#### Some rules are changed:
+
+- [`no-cond-assign`](http://eslint.org/docs/rules/no-cond-assign) now follows the `airbnb` style to always be an error
+- [`import/no-duplicates`](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-duplicates.md) error instead of warning
+- [`import/prefer-default-export`](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/prefer-default-export.md) error instead of warning
+- [`react/prop-types`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/prop-types.md) error instead of warning
+
+#### Some rules are now checked by [eslint-plugin-prettier](https://github.com/prettier/eslint-plugin-prettier)
+
+- `dot-location`
+- `no-multi-spaces`
+- `wrap-iife`
+- `comma-dangle`
+- `no-extra-semi`
+- `array-bracket-spacing`
+- `block-spacing`
+- `brace-style`
+- `comma-spacing`
+- `comma-style`
+- `computed-property-spacing`
+- `eol-last`
+- `func-call-spacing`
+- `indent`
+- `jsx-quotes`
+- `key-spacing`
+- `keyword-spacing`
+- `new-parens`
+- `newline-per-chained-call`
+- `no-mixed-spaces-and-tabs`
+- `no-multiple-empty-lines`
+- `no-spaced-func`
+- `no-trailing-spaces`
+- `no-whitespace-before-property`
+- `object-curly-spacing`
+- `object-property-newline`
+- `one-var-declaration-per-line`
+- `padded-blocks`
+- `quote-props`
+- `semi`
+- `semi-spacing`
+- `space-before-blocks`
+- `space-before-function-paren`
+- `space-in-parens`
+- `space-infix-ops`
+- `space-unary-ops`
+- `unicode-bom`
+- `arrow-parens`
+- `arrow-spacing`
+- `generator-star-spacing`
+- `rest-spread-spacing`
+- `template-curly-spacing`
+- `yield-star-spacing`
+- `jsx-closing-bracket-location`
+- `jsx-curly-spacing`
+- `jsx-indent-props`
+- `jsx-max-props-per-line`
+- `jsx-space-before-closing`
+- `jsx-wrap-multilines`
+- `jsx-first-prop-new-line`
+- `jsx-equals-spacing`
+- `jsx-indent`
+- `flowtype/space-after-type-colon`
+- `flowtype/space-before-type-colon`
+
+#### Some rules are special (see [eslint-config-prettier#special-rules](https://github.com/prettier/eslint-config-prettier#special-rules))
+
+- `max-len` `'error'` to give hints about strings that might need splitting
+- `no-mixed-operators` `off` because `prettier` formats those non-trivial
+
+### Open Questions that need an answer
+
+- `no-mixed-operators` should we turn this rule on somehow?
+- [`no-use-before-define`](http://eslint.org/docs/rules/no-use-before-define) was turned to `error` (old config does not work anymore, any change needed?)
+- were `"jsx-a11y/onclick-has-focus": "off"` and `"jsx-a11y/onclick-has-role": "off"` replaced with `"jsx-a11y/interactive-supports-focus": "error"`?
```

Comment:
I'd say, upgrading to latest airbnb config is definitely good. There's no need to discuss any of the changes coming in with this update here. If we find some of the new rules to cause troubles for us, we can still discuss changing or disabling them in separate PR discussions.